scripts/get-coreos

```diff
@@ -1,13 +1,14 @@
 #!/bin/bash
 # USAGE: ./scripts/get-coreos
-# USAGE: ./scripts/get-coreos channel version dest
+# USAGE: ./scripts/get-coreos channel version dest oem_id
 set -eou pipefail
 
 GPG=${GPG:-/usr/bin/gpg}
 
 CHANNEL=${1:-"stable"}
 VERSION=${2:-"1298.7.0"}
 DEST_DIR=${3:-"$PWD/examples/assets"}
+OEM_ID=${4:-""}
```

Comment:
Rather than adding more positional arguments, I think this one needs to be an optional argument